src/main/java/technology/dice/dicefairlink/AuroraReadEndpoint.java

```diff
@@ -22,43 +22,62 @@
 import java.time.Duration;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
 import java.util.NoSuchElementException;
 import java.util.Optional;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
-public class AuroraReadonlyEndpoint {
-  private static final Logger LOGGER = Logger.getLogger(AuroraReadonlyEndpoint.class.getName());
+public class AuroraReadEndpoint {
```

Comment:
This is still readonly. The fact that we use the master database to perform reads does not change the nature of the connections.

Reply:
Fixed.